src/Nova/EscaperoomSlot.php

```diff
@@ -86,10 +87,9 @@ public function fields(Request $request)
                 return $this->formatted_start;
             })->exceptOnForms(),
             DateTime::make('Start At')->required(),
-            nova('rate') ? BelongsTo::make('Rate', 'rate', nova('rate'))->hideWhenCreating()->nullable() : null,
+            //TODO is missing the morph relation with schedule_id and schedule_type and the models
```

Comment:
@drewroberts what will be the models for this `morph` relation?

Reply:
@pdbreen Could you start taking a look at this package and give us some direction on changes you want to make and how to get it structured?

Reply:
@wolfrednicolas here's an example of how to setup morph relationships in Nova without knowing which packages will be installed:  https://github.com/tipoff/notes/blob/f20b9ed1dbcd98d49e3d479f4c5bd830e6787f19/src/Nova/Note.php#L48

```
            MorphTo::make('Noteable')->types(array_filter([
                nova('user'),
                nova('contact'),
                nova('customer'),
                nova('order'),
                nova('booking'),
                nova('game'),
                nova('block'),
                nova('slot'),
            ])),
```


@drewroberts - it looks like tipoff/addresses needs an issue created to fix the Nova `addressable` morph to use this technique.  The addressable type list should include cart, order, voucher, and probably a bunch of other things, but it needs to be done as above so the packages remain optional.

Reply:
@joshtorres @phuclh Could you create an issue for this and get this implemented in the addresses package?

Reply:
Thank you @pdbreen I replicate the same structure here.  👍